lib/rules/rule-nested-empty-line-before/__tests__/index.js

```diff
@@ -122,6 +122,50 @@ testRule(rule, {
 
 testRule(rule, {
   ruleName,
+  config: [ "always", { except: ["after-rule"] } ],
+
+  accept: [ {
+    code: "a {} b {}",
+    description: "non-nested node ignored",
+  }, {
+    code: "a {}\nb {}",
+    description: "non-nested node ignored",
+  }, {
+    code: "a {\n color: pink; \n\n b {color: red; } \n c {color: blue; }\n}",
+    description: "css property",
+  }, {
+    code: "a {\n $var: pink; \n\n b {color: red; } \n c {color: blue; }\n}",
+    description: "scss variable",
+  }, {
+    code: "a {\n --custom-prop: pink; \n\n b {color: red; } \n c {color: blue; }\n}",
+    description: "custom property",
+  }, {
+    code: "a {\n @media {\n\n   b {}\n }\n\n c {}\n d {}\n}",
+    description: "media rule",
+  }, {
+    code: "a {\n color: pink; \r\n\r\n b {color: red; } \n c {color: blue; }\n}",
+    description: "CRLF",
+  } ],
+
+  reject: [ {
+    code: "a {\n color: pink; b {color: red; }\n c {color: blue; }\n}",
+    message: messages.expected,
+  }, {
+    code: "a {\n color: pink;\n b {color: red; }\n c {color: blue; }\n}",
+    message: messages.expected,
+  }, {
+    code: "a {\n @media {\n\n   b {}\n }\n c {}\n d {}\n}",
+    description: "media rule",
+    message: messages.expected,
+  }, {
+    code: "a {\r\n color: pink;\r\n b {\rcolor: red; \r}\r\n c {\rcolor: blue; \r}\r\n}",
```

Comment:
There's three `\r`'s here unaccompanied by `\n`'s. Can you add the `\n`'s please.